### ISSUE TITLE
Problem with %superior_island_top_level_position% 

### ISSUE THREAD

**Issue by Chrqmated** [BG-Software-LLC/SuperiorSkyblock2#533]
### Minecraft's Version

1.12.2 Paper

### Plugin's Version

Latest dev build version of SuperiorSkyblock2
(i have tested with stable version, same problem)

### Describe the bug

A. I have 2 accounts, the first account have 200 levels (/is level)
B. my other accounts have 100 levels (/is level)

i use the command /papi parse me %superior_island_top_level_position% (i'm 2 in my other account (B)

i place 500 levels with my other account (B), i use command /is top, i'm first yeahh !! but, i use /papi parse me %superior_island_top_level_position% i'm 2 (noooo :/)

i use the command /is admin reload, i'm always 2, i try restart the server, after the restart, i use the command /papi parse me %superior_island_top_level_position% and i'm 1

In summary, the placeholder% superior_island_top_level_position% does not update automatically, it must be restarted ...

### To Reproduce

A. I have 2 accounts, the first account have 200 levels (/is level)
B. my other accounts have 100 levels (/is level)

i use the command /papi parse me %superior_island_top_level_position% (i'm 2 in my other account (B)

i place 500 levels with my other account (B), i use command /is top, i'm first yeahh !! but, i use /papi parse me %superior_island_top_level_position% i'm 2 (noooo :/)

i use the command /is admin reload, i'm always 2, i try restart the server, after the restart, i use the command /papi parse me %superior_island_top_level_position% and i'm 1

**Comment by OmerBenGera** [comment]
Are you sure it's not something with Papi somehow caches data?

**Comment by OmerBenGera** [comment]
I've just tested it and it works fine for me.

**Comment by OmerBenGera** [comment]
Make sure you click on the "sort by levels" so it recalculates the island positions by level and not by worth.

**Comment by OmerBenGera** [comment]
Any news regarding this?
